index.html

```diff
@@ -25,6 +25,110 @@ <h1>I'm <b>Behnam</b>.<span> Welcome to my website!</span> </h1>
                     <li><a href="https://github.com/Behnam1369" class="social"><i class="fa fa-2x fa-github"></i></a></li>
                 </ul>
             </header>
+            <div class="seprator1"></div>
+            <section id="works">
+                <div class="work">
+                    <img src="images/work1.jpg" alt="Work 1"/>
+                    <h3>My Gym Mate</h3>
+                    <span class="span1">Athletico</span>
+                    <i class="devider"></i>
+                    <span class="span2">Full Stack Developer</span>
+                    <i class="devider"></i>
+                    <span class="span2">2018</span>
+                    <p>
+                        A web application which help people to find partners for thair sport activities. 
+                        You can simply search for partners, get connected to them and enjoy your sport activities together.
+                    </p>
+                    <span class="tag">html</span>
+                    <span class="tag">css</span>
+                    <span class="tag">javascript</span>
```

Comment:
- [ ] Please, make sure to use `<ul>` tags for your languages here and the correspondent `<li>` tag as well. This is really helpful to make your code more semantic, so later on, whenever you (or any other developer) see your project you can understand it without struggling.